CONTRIBUTING.md

```diff
@@ -22,3 +22,54 @@ use GitHub pull requests for this purpose.
 ### The small print
 Contributions made by corporations are covered by a different agreement than
 the one above, the Software Grant and Corporate Contributor License Agreement.
+
+### Configuring a local development environment
+This can optionally done within a [`virtualenv`](https://virtualenv.pypa.io/en/latest/).
+
+### Fork the repository
```

Comment:
Something seems to have gone wrong here with a merge - the part following from here on is already in the upstream file, but is showing as an addition in "Files changed".